src/cromshell/metadata/command.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+import click
+import requests
+
+from cromshell.utilities import cromshellconfig, http_utils, io_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="metadata")
+@click.argument("workflow_id")
+# Setting is_flag=False, flag_value=value tells Click that the
+# option can still be passed a value, but if only the flag is
+# given the flag_value is used.
+@click.option(
+    "--slim-metadata",
```

Comment:
If we're changing this from a top level command lets make it shorter.  Like just "slim".

Reply:
Or maybe rename to "keys" or something.

Reply:
--slim "id,executionStations,..."

Reply:
changed to key